mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/refusjon/task/JournalforRefusjonskrav.kt

```diff
@@ -63,6 +67,11 @@ class JournalforRefusjonskrav(
         requireNotNull(krav) { "Fant ikke refusjonskrav med id=$id" }
         require(krav.status == RefusjonskravStatus.GODKJENT_AV_ARRANGOR) { "Krav må være godkjent" }
 
+        val gjennomforing = tiltaksgjennomforingRepository.get(krav.gjennomforing.id)
+        requireNotNull(gjennomforing) { "Fant ikke gjennomforing til refusjonskrav med id=$id" }
+        val fagsakId = gjennomforing.tiltaksnummer ?: gjennomforing.lopenummer
```

Comment:
Jeg regner med dette er sånn vi vil gjore det. Bruke tiltaksnummer for de som har blitt skrevet til arena og allerede blitt behandlet der potensielt.

Og lopenummer på de nye etter vi slutter og skrive tilbake til Arena. Den dagen så regner jeg med vi begynner å vise lopenummeret i admin-flate (hvis det ikke er fallback dit allerede?)